### PR TITLE
fix/PLF-8383 : Adjust error message position

### PR DESCRIPTION
**Pull request by aelbenney**

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less [hunk 1]
```diff
@@ -314,11 +315,15 @@
                         border-radius: 5px;
                         border: 3px dashed #e6e6e6;
                         font-size: 14px;
-                        height: 100%;
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less]: We still rely on fixed height/margin like here or https://github.com/exodev/platform-ui/blob/7638544c85b3006c331010f41bb67be9376738fe/platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less#L273 or https://github.com/exodev/platform-ui/blob/7638544c85b3006c331010f41bb67be9376738fe/platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less#L277. We should not IMO, flex allows to get rid of this.

File: platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less [hunk 2]
```diff
@@ -1250,4 +1200,10 @@
...
   }
-}
\ No newline at end of file
+}
+
+@media screen and (max-width: 1025px) {
+					#newsActivityComposer #newsForm .dropZone img {
+                        height: 80px;
+                }
+            }
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less]: Indentation is not good.

File: platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less [hunk 3]
```diff
@@ -91,6 +91,135 @@
...
+                        }
+                        .abortFile {
+                            height: 0;
+                        }
+                    }
+                    #newsSummary {
+                        margin-top: 10px;
+                    }
+                    #UINewsSummaryDescription {
+                        color: #585858;
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less]: Don't we have a variable for this color ? (to check for others colors)

File: platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less [hunk 4]
```diff
@@ -91,6 +91,135 @@
             .fileSize {
                 width: 75px;
             }
+
+            #newsActivityComposer {
```
Comment (thomasdelhomenie) [inline platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less]: This block is in `.uploadContainer`, it should not IMO. The style is not applied to the form. Here is what I see:
image

File: platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less (hunk 2 above)
Comment (SaraBoutej) [reply platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less]: Done!

File: platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less (hunk 3 above)
Comment (SaraBoutej) [reply platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less]: Done!

File: platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less (hunk 4 above)
Comment (SaraBoutej) [reply platform-ui-skin/src/main/webapp/skin/less/social/portlets/uiActivityStreamPortlet/Style.less]: Done!

**Pull request merged**